static/app/utils/discover/discoverQuery.spec.jsx

```diff
@@ -25,9 +25,9 @@ describe('DiscoverQuery', function () {
   describe('with eventsv2', function () {
```

Comment:
This is still referencing `eventsv2`, is there already an `events` test block here with duplicate coverage and thus we can delete this?